plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -521,8 +521,7 @@ function MosaicMenuItem:update()
             },
         }
     else
-        local filemanager = require("apps/filemanager/filemanager").instance
-        local file_selected = filemanager.selected_files and filemanager.selected_files[self.filepath]
+        local file_selected = self.menu.filemanager.selected_files and self.menu.filemanager.selected_files[self.filepath]
```

Comment:
In History/Favorites, I guess self.menu.filemanager is nil - so "can't index a nil value" will happen when accessing self.menu.filemanager.selected_files ?